src/plone/restapi/indexers.py

```diff
@@ -2,7 +2,7 @@
 # This is an experimental feature meant for use in Volto only!
 # This code is likely to change in the future, even within minor releases.
 # We will make sure plone.restapi latest always works with the latest Volto release.
-# This code is planned to being refactored into CMFPlone 6.0 as soon as Volto 4 final is out.
```

Comment:
@ericof haha. That comment did not age well...

Reply:
That is why I "edited" it.